src/engine/sync/enginesync.cpp

```diff
@@ -371,7 +371,7 @@ void EngineSync::notifyPlayingAudible(Syncable* pSyncable, bool playingAudible)
             // Even if we didn't change leader, if there is only one player (us), then we should
```

Comment:
The comment doesn't match the new method invocation.

Reply:
fixed